src/Test.UnitTests.Sarif.Multitool/Test.UnitTests.Sarif.Multitool.csproj

```diff
@@ -15,13 +15,18 @@
 
   <ItemGroup>
     <None Remove="TestData\ConvertCommand\SemmleQlSample.csv" />
+    <None Remove="TestData\ExportRuleDocumentationCommand\ExpectedOutputs\Test.md" />
     <None Remove="TestData\MergeCommand\ExpectedOutputs\NoInputFiles.sarif" />
     <None Remove="TestData\PageCommand\elfie-arriba - Copy.sarif" />
     <None Remove="TestData\PageCommand\elfie-arriba.sarif" />
     <None Remove="TestData\RebaseUriCommand\ExpectedOutputs\RunWithArtifacts.sarif" />
     <None Remove="TestData\RebaseUriCommand\RunWithArtifacts.sarif" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Include="TestData\ExportRuleDocumentationCommand\ExpectedOutputs\Test.md" />
```

Comment:
>Test [](start = 87, length = 4)

I know that VS added this `EmbeddedResource` element. But just for neatness, would you please move it into the group below, just below Line 31 (in alphabetical order, in the same group, with the other command test resources)? Thanks.